GG/src/DropDownList.cpp

```diff
@@ -118,6 +121,9 @@ class ModalListPicker : public std::enable_shared_from_this<ModalListPicker>, pu
     const size_t        m_num_shown_rows;
     const DropDownList* m_relative_to_wnd;
     bool                m_dropped; ///< Is the drop down list open.
+
+    /** Should the list wnd scroll when dropped? */
```

Comment:
I fixed the comment, but left the variable name as is since that reflects its current usage.

We can change the variable name when we change the code, after the production window bug(s) is fixed.

Keep the original issue open and perhaps copy your comment into the original issue as a reminder.

I'd prefer shift-mousewheel in this case since it modifies the UI behavior, while still doing the same action.

By analogy with a keyboard and the 'q' key: `q` types a lower case 'q', `shift-q`, types an uppercase 'Q' and `ctrl-q` quits the application.  Both `q` and `shift-q` enter a letter into the data stream, the same action.  `ctrl-q` does a different action.  

In our case both `mousewheel` and `shift-mousewheel` scroll something, the whole list or the embedded llist.  I would expect `ctrl-mousehweel` to do something different, like maybe a scrolling delete.